components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManager.java

```diff
@@ -136,17 +136,18 @@ List<BasicOrganization> getOrganizations(Integer limit, String after, String bef
     /**
      * List or search organizations authorized for the user.
      *
-     * @param limit     The maximum number of records to be returned.
-     * @param after     The pointer to next page.
-     * @param before    The pointer to previous page.
-     * @param sortOrder The sort order, ascending or descending.
-     * @param filter    The filter string.
-     * @param recursive Determines whether records should be retrieved in a recursive manner.
+     * @param limit               The maximum number of records to be returned.
+     * @param after               The pointer to next page.
+     * @param before              The pointer to previous page.
+     * @param sortOrder           The sort order, ascending or descending.
+     * @param filter              The filter string.
+     * @param recursive           Determines whether records should be retrieved in a recursive manner.
+     * @param applicationAudience The application audience.
      * @return the list of {@link BasicOrganization}s.
      * @throws OrganizationManagementException The exception thrown when listing organizations.
      */
     List<BasicOrganization> getUserAuthorizedOrganizations(Integer limit, String after, String before, String sortOrder,
-                                                           String filter, boolean recursive)
+                                                           String filter, boolean recursive, String applicationAudience)
```

Comment:
let's introduce a new method without changing the exiting method

Reply:
Fixed